### PR TITLE
Alma Linux 8 and CentOS Stream 8 compatibility 

### PR DESCRIPTION
**Pull request by justinswe**

### CODE REVIEW

File: test/cookbooks/scponly-test/attributes/default.rb
Comment (justinswe) [inline test/cookbooks/scponly-test/attributes/default.rb:1]: The key is giving the following cookstyle error which I'm not sure how we should proceed to resolve.
"Chef/Security/SshPrivateKey: Do not include plain text SSH private keys in your cookbook code."

File: test/cookbooks/scponly-test/recipes/scponly_chroot.rb [hunk 1]
```diff
@@ -1,9 +1,15 @@
+include_recipe 'osl-selinux'
+
 scponly_user 'scponly_test_chroot' do
   write_dir 'write'
   public_key node['scponly']['public_key']
   chroot true
 end
 
+selinux_fcontext '/var/lib/chroots/home/scponly_test_chroot/.ssh/id_rsa-scponly_user-scponly_test_chroot' do
+  secontext      'httpd_sys_rw_content_t'
```
Comment (justinswe) [inline test/cookbooks/scponly-test/recipes/scponly_chroot.rb]: I'm not sure if the secontext I set on the private key is what should be used. However, as it currently stands it passes the tests where before it was failing.

File: test/cookbooks/scponly-test/attributes/default.rb
Comment (justinswe) [reply test/cookbooks/scponly-test/attributes/default.rb:1]: Resolved with commit.

File: test/cookbooks/scponly-test/recipes/scponly_chroot.rb (hunk 1 above)
Comment (ramereth) [reply test/cookbooks/scponly-test/recipes/scponly_chroot.rb]: I believe `ssh_home_t` is better. Can you please change that?

Comment (justinswe) [reply test/cookbooks/scponly-test/recipes/scponly_chroot.rb]: Yes just made the changes to the context.

**Review by ramereth — APPROVED** [review]
Patch bump

**Comment by justinswe** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.5.1.  Have a nice day!

**Pull request merged**